docs/source/reference/storage.rst

```diff
@@ -117,24 +136,31 @@ its performance requirements and size of the data.
    * -
      - .. centered:: :code:`mode: MOUNT`
      - .. centered:: :code:`mode: COPY`
+     - .. centered:: :code:`mode: MOUNT_CACHED`
    * - Best for
-     - Writing task outputs (e.g., checkpoints, logs); reading very large data that won't fit on disk.
+     - Writing task outputs; reading very large data that won't fit on disk.
      - High performance read-only access to datasets that fit on disk.
+     - High performance writes (e.g., model checkpoints, logs) that fit on disk cache.
    * - Performance
      - |:yellow_circle:| Slow to read/write files. Fast to provision.
      - |:white_check_mark:| Fast file access. Slow at initial provisioning.
+     - |:white_check_mark:| Fast file access. Fast at initial provisioning.
    * - Writing to buckets
      - |:yellow_circle:| Most write operations [1]_ are supported.
      - |:x:| Not supported. Read-only.
+     - |:white_check_mark:| All write operations are supported. [2]_
    * - Disk Size
-     - |:white_check_mark:| No disk size requirements [2]_ .
+     - |:white_check_mark:| No disk size requirements [3]_ .
      - |:yellow_circle:| VM disk size must be greater than the size of the bucket.
+     - |:yellow_circle:| No disk size requirements, but cached data needs to fit on disk. 
 
 .. [1] ``MOUNT`` mode does not support the full POSIX interface and some file
     operations may fail. Most notably, random writes and append operations are
     not supported.
 
-.. [2] Disk size smaller than the object size may cause performance degradation
+.. [2] In ``MOUNT_CACHED`` mode, writes are not immediately consistent across multiple nodes.
```

Comment:
Add something like "See MOUNT_CACHED mode section for more detail" with a link to the dedicated section.